libnetwork/network.go

```diff
@@ -455,7 +454,6 @@ func (n *Network) CopyTo(o datastore.KVObject) error {
 	dstN.enableIPv4 = n.enableIPv4
 	dstN.enableIPv6 = n.enableIPv6
 	dstN.persist = n.persist
-	dstN.postIPv6 = n.postIPv6
```

Comment:
I'm wondering if this could cause issues when downgrading? Maybe we should keep this field for one more major release (or until the latest Mirantis release to support that behavior reaches EOL)?

Reply:
It works out ok, because the default bridge gets set up again by the old build.

(We don't support `live-restore` over a major version, but that happens to be fine too. Running containers keep their addresses on upgrade anyway, containers created after upgrade get IPAM addresses, and keep them on live-downgrade.)